updater/spec/dependabot/api_client_spec.rb

```diff
@@ -199,7 +202,11 @@
           to(have_requested(:post, create_pull_request_url).
              with do |req|
                data = JSON.parse(req.body)["data"]
-               expect(data["grouped-update"]).to be true
+               # FIXME: Once we fully remove the `group-all` prototype behavior, we can also remove the
+               # `grouped-update` expectation since dependency-groups will require rules and only
+               # return the `dependency-group` key
+               expect(data["grouped-update"]).to be nil
```

Comment:
Hmm, this comment suggests should we still be sending this key as `true` if a group is involved?

I don't think we need to care about this key anymore even if the placeholder is involved though. I believe the API behaviour will set the `grouped-update` flag on its side if a Dependency Group is sent, even if that group isn't in the database. This should result in jobs carried out with a placeholder that was either injected by the API or by the Updater itself having the same behaviour.